shortbread-website/content/customizing/_index.md

```diff
@@ -0,0 +1,141 @@
++++
+title = "Customizing Shortbread"
```

Comment:
Right now this document is a mix of custom tiles (i.e. replacing shortbread), adding a set of tiles on top (keeping shortbread the same), custom styles (not changing anything), and generating your own shortbread tiles.

It's not clear which parts of the document are aimed at which purpose. For creating non-shortbread tiles I think pointing at the tools for generating your own shortbread tiles is enough. People can take them and modify the output tiles. Tiles on top doesn't really need an example as all the different parts are non-shortbread. The other two could each use their own page.

Reply:
The goal of this page is to inform people about the ways to do Shortbread mashups. The concrete examples are for customizing Shortbread itself, some chapters discuss ways to combine Shortbread with additional tilesets. I think, a page like this should exist. Maybe we should just change the title?

Reply:
I'm okay with it being on one page if there's more clarity what each part of the page is talking about.

Reply:
Should be clear now.